### DISCUSSION TITLE
アバターが変更できなくなった

### DISCUSSION THREAD

**Discussion by arch-herobrine** [vercel/community#5194] [Bug Report]
### Question

アバターを設定しようとした時、「Avatar has been updated.」の表示が出たにも関わらずアバターが更新されませんでした(DevToolを見たところHTTP 508エラーを返していた)。
 
それどころかアバターがリンク切れを起こしてしまい(DevToolで見たところHTTP 400エラーを返していた)途方に暮れています。
 
だれか対処法を教えてください。
 
追記: 本当は英語で書くべきだったのだろうけど機械翻訳で元の文のニュアンスとずれるのも嫌だったのでそのまま書きました

**Comment by EdamAme-x** [comment]
再現手順をもう少し詳しくお願いします。

**Reply by arch-herobrine** [reply]
再現手順と言われてもただアイコンを設定するだけなので...
 
一応スクリーンショット置いときます
 
![image](https://github.com/vercel/community/assets/112319383/ccb35c2e-6edf-468a-babe-b69feebe4c13)

**Reply by EdamAme-x** [reply]
私の環境でも確認出来ました。
(safari)

**Reply by arch-herobrine** [reply]
これってVercel側のバグなのでしょうか...?
 
それともまだ報告されてない障害だったりするのかな

**Reply by amyegan** [reply]
Thanks for reporting this. It does look like a bug. I will share this info with with the engineering team and I'll post updates here when we know more about the cause

**Reply by EdamAme-x** [reply]
ty

**Reply by amyegan** [reply]
Avatar selection should be working normally now. Please let me know if you still have any trouble with it. 🙇

**Reply by EdamAme-x** [reply]
ty
it works!
<img width="95" alt="image" src="https://github.com/vercel/community/assets/121654029/2d51fdfa-8a65-4d5b-ab97-25219fa615cd">
